Scripts/Entities/Player/PlayerCommand.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Sankari;
+
+public abstract class PlayerCommand : EntityCommand
```

Comment:
I would make this
```suggestion
public abstract class PlayerCommand : EntityCommand<Player>
```

and remove the Player property.

Reply:
I'll leave that up to you unless if you want me to convert the player commands back to entity commands and do this suggestion I can do it but I would prefer you do it.

Reply:
As long as you revert the Dash (and maybe Wall Jump) back to EntityCommands, I could handle the new commands.

Reply:
The old dash command code had bugs, the way it is now, those bugs are gone so I'm not sure how I would revert it?